snippets/nrf70-driver-debug/overlay-nrf70-driver-debug.conf

```diff
@@ -0,0 +1,23 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_SHELL=y
+CONFIG_SHELL_STACK_SIZE=4600
+CONFIG_NET_SHELL=y
+CONFIG_SHELL_GETOPT=y
+CONFIG_SHELL_CMDS_RESIZE=n
+CONFIG_NRF700X_UTIL=y
+CONFIG_NET_L2_WIFI_SHELL=y
+CONFIG_NET_STATISTICS=y
+CONFIG_NET_STATISTICS_WIFI=y
+CONFIG_NET_STATISTICS_USER_API=y
+CONFIG_SYS_HEAP_RUNTIME_STATS=y
+CONFIG_LOG=y
+CONFIG_PRINTK=y
+CONFIG_LOG_MODE_IMMEDIATE=y
+CONFIG_NRF_WIFI_RPU_RECOVERY=y
+CONFIG_WIFI_NRF700X_LOG_LEVEL_DBG=y
+CONFIG_NRF_WIFI_LOG_LEVEL_DBG=y
```

Comment:
These suffer from the same problem as WPA supplicant debugs, too verbose and memory consuming. But on the other hand, don't want to end up with too many snippet variants, may be, below?

1. WPA supplicant debug
2. Driver debug
3. Driver verbose debug (enable BUG debug as well)